index.html

```diff
@@ -0,0 +1,315 @@
+            remote work.
+            <br />
+            I've also been a good helping hand to a lot of developers in
+            different communities!
+            <br />
+            if you'd like to get in contact about potential job opportunities,
+            please reach out!
+          </p>
+          <a
+            href="https://docs.google.com/document/d/1lYq0Kb50TujzJzDWGobIchRI17JMnAiZMF9OBQgNU68/edit?usp=sharing"
+            class="link"
+            target="_blank"
+            rel="noreferrer noopener"
+          >
+            Get my resume
+          </a>
+          <img class="about-me-icon" src="./media/svg/icon-2.svg" alt="Icon" />
+        </section>
+        <hr />
+        <div class="about-me-section stacks">
+          <ul>
+            <li class="languages list-title">
+              Languages
+              <ul>
+                <li>JavaScript</li>
+                <li>TypeScript</li>
+                <li>Ruby</li>
+                <li>HTML</li>
+                <li>CSS</li>
+                <li>Sass</li>
+              </ul>
+            </li>
+          </ul>
+          <ul>
+            <li class="frameworks list-title">
+              Frameworks
+              <ul>
+                <li>React & Redux</li>
+                <li>Material-UI</li>
+                <li>Jest</li>
+                <li>Bootstrap</li>
+                <li>Ruby on Rails</li>
+                <li>RSPec</li>
+              </ul>
+            </li>
+          </ul>
+          <ul>
+            <li class="skills list-title">
+              Skills
+              <ul>
+                <li>GitHub</li>
+                <li>CodePen</li>
+                <li>GitLab</li>
+                <li>Terminal</li>
+                <li>WSL</li>
+                <li>Remote Pair-Programming</li>
+                <li>Teamwork</li>
+                <li>Mentoring</li>
+                <li>Code Reviews</li>
+                <li>Mobile/Responsive Development</li>
+              </ul>
+            </li>
+          </ul>
+        </div>
+      </div>
+    </main>
+    <div class="contact-container">
+      <div class="contact-left">
+          <h2 class="contact-title">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
+      </div>
+      <div class="contact-right">
+          <form action="https://formspree.io/f/xbjqgnpd" method="POST" class="contact-form">
+            <div class="fullname">
+              <input type="text" id="firstname" name="fullname" placeholder="Email address"  maxlength="30" required>
+              <input type="text" id="lastname" name="fullname" placeholder="Email address"  maxlength="30" required>
+            </div>
+              <input type="email" id="email" name="email" placeholder="Email address" maxlength="30" required>
+              <textarea id="message" name="message" placeholder="Hello Miguel! Glad to see you" maxlength="30" required></textarea>
```

Comment:
- [ ] When working with textareas we want to make sure the User is able to write a enough amount of text, I suggest we change the `maxlength=30` to `maxlength=500`